File: Web/RMA-Web-Shell/projects/clientcare/src/app/lead-manager/models/lead-details.ts
import { ClientTypeEnum } from '../../policy-manager/shared/enums/client-type-enum';
import { LeadClientStatusEnum } from '../../policy-manager/shared/enums/leadClientStatusEnum';

export class LeadDetails {
    leadId: number;
    dateCreated: Date;
    leadStatus: LeadClientStatusEnum;
    memberName: string;
    clientType: ClientTypeEnum;
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Lead/LeadDetails.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

using System;

namespace RMA.Service.ClientCare.Contracts.Entities.Lead
{
    public class LeadDetails
    {
        public int LeadId { get; set; }
        public DateTime DateCreated { get; set; }
        public LeadClientStatusEnum LeadStatus { get; set; }
        public string MemberName { get; set; }
        public ClientTypeEnum ClientType { get; set; }
        public DateTime SLA { get; set; }
        public int ProductsInterestedCount { get; set; }
    }
}
